pages/search/search.jsx

```diff
@@ -56,7 +57,7 @@ class Search extends React.Component {
   updateBrowserHistory() {
     let keywordSearched = this.state.keywordSearched;
     let helpType = this.state.helpType;
-    let location = { pathname: this.props.location.pathname };
+    let location = { pathname: "/search" };
```

Comment:
...wanted to make sure old link work which currently follow the `/search?...` path. Let me know if there's a better way to do this 🤔